modules/ROOT/pages/sharing-applications.adoc

```diff
@@ -0,0 +1,32 @@
+= Share Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: sharing applications, sharing,
+
+When working on a team, a Mule runtime engine project increases in size, increases in its number of developers, and must run in other environments, such as test and production.
+
+Sharing applications is an opportunity for splitting work across teams, whether these teams work on the same overarching project or on different projects with an accent put on reuse.
+Applications are self-contained archives and can therefore be shared via a simple file or web server.
+
+To share applications follow the next steps:
+
+. You must document the name and content of the properties file that each application is expecting to use.
+
+[NOTE]
+Make sure to encrypt your properties file as detailed in xref:secure-configuration-properties.adoc[Secure Configuration Properties]. Keep the decryption key of the properties file secure so you can pass it to the administrator(s) who deploys and runs your Mule application.
```

Comment:
Usually, developers would use a non-encrypted files (or encrypted but entirely dev files). The creation and encryption of the final versions should be done directly by the admins. However, this is also correct depending on the scenario

Reply:
Updated